tests/container/base/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM fedora:34
```

Comment:
We just switched away from using Fedora as the base image in a recent PR. It may be better to keep the tooling consistent, especially since Fedora moves so fast.

Reply:
Oh, cool. I'll switch to CentOS 8.

Reply:
Done :)